Apps/Examples/Examples/All Examples/Localization/LocalizationSettingsExample.swift

```diff
@@ -0,0 +1,175 @@
+            worldviewButtonTitle.append(": \(worldview)")
+        }
+
+        languageButton.setTitle(languageButtonTitle, for: .normal)
+        worldviewButton.setTitle(worldviewButtonTitle, for: .normal)
+    }
+
+    // MARK: - Actions
+
+    @objc private func languageButtonPressed(_ sender: UIButton) {
+        let alert = UIAlertController(title: "Languages",
+                                      message: "Please select a language to localize to.",
+                                      preferredStyle: .actionSheet)
+        let locale = Locale.current
+        let settings = settings
+        // get the selected language setting(BCP-47 language tag, e.g. "en")
+        let selectedLanguage = try? settings.get(key: MapboxCommonSettings.language, type: String.self).get()
+
+        supportedLanguages.map { language -> UIAlertAction in
+            let title = locale.localizedString(forIdentifier: language)?
+                .appending(language == selectedLanguage ? " ✓" : "")
+
+            return UIAlertAction(title: title, style: .default) { _ in
+                do {
+                    // set the language setting according to the language selected by the user
+                    try settings.set(key: MapboxCommonSettings.language, value: language).get()
+                    sender.setTitle("Language: \(language)", for: .normal)
+                } catch {
+                    print("Failed to set the language, error: \(error.localizedDescription)")
+                }
+            }
+        }.forEach(alert.addAction(_:))
+
+        alert.addAction(UIAlertAction(title: "Cancel", style: .cancel))
+
+        present(alert, animated: true)
+    }
+
+    @objc private func worldviewButtonPressed(_ sender: UIButton) {
+        let alert = UIAlertController(title: "Worldviews",
+                                      message: "Please select a worldview filter.",
+                                      preferredStyle: .actionSheet)
+        let locale = Locale.current
+        let settings = settings
+        // get the selected worldview setting(ISO 3166-1 alpha-2 code, e.g. "US")
+        let selectedWorldview = try? settings.get(key: MapboxCommonSettings.worldview, type: String.self).get()
+
+        supportedWorldviews.map { worldview -> UIAlertAction in
+            let title = locale.localizedString(forRegionCode: worldview)?.appending(worldview == selectedWorldview ? " ✓" : "")
+
+            return UIAlertAction(title: title, style: .default) { _ in
+                do {
+                    // set the worldview setting according to the language selected by the user
+                    try settings.set(key: MapboxCommonSettings.worldview, value: worldview).get()
+                    sender.setTitle("Worldview: \(worldview)", for: .normal)
+                } catch {
+                    print("Failed to set the worldview, error: \(error.localizedDescription)")
+                }
+            }
+        }.forEach(alert.addAction(_:))
```

Comment:
[discussion, nit] 
It's probably just me, but I'm not a big fun of the combination `map + forEach`.
Wouldn't common for-in look a bit cleaner?